minvws/nl-kat-coordination
# Remove source link in Findings Report when source is none

### Changes
This PR removes the link from the Occurrence's source when the source is None.

### Issue link
Closes #3636

### Demo
Before:
![afbeelding](https://github.com/user-attachments/assets/e0880ab6-b0ac-4fe7-a488-0ed3accec338)


After:
![afbeelding](https://github.com/user-attachments/assets/351bbfe8-8955-43df-9184-5370069d7e47)


### QA notes
- Create a Findings Report (one with a KAT-NAMESERVER-NO-IPV6 or KAT-NO-DNSSEC finding for example)
- Expand the table rows and check for a source == None
- The link should be removed
- The link for sources that are not None should still be visible and working

---

### Code Checklist







---

## Checklist for code reviewers:

Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_code.md) into your comment.

---

## Checklist for QA:

Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_qa.md) into your comment.
